packages/suite-base/src/panels/PieChart/PieChart.tsx

```diff
@@ -0,0 +1,279 @@
+// SPDX-FileCopyrightText: Copyright (C) 2024-2025 Yukihiro Saito <yukky.saito@gmail.com>
+// SPDX-FileCopyrightText: Copyright (C) 2025 Takayuki Honda <takayuki.honda@tier4.jp>
+// SPDX-License-Identifier: MPL-2.0
+
+import * as _ from "lodash-es";
+import { useCallback, useEffect, useLayoutEffect, useReducer, useState, useMemo } from "react";
+import {
+  PieChart as RechartsPieChart,
+  Pie,
+  Cell,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+
+import { parseMessagePath } from "@lichtblick/message-path";
+import { PanelExtensionContext, SettingsTreeAction } from "@lichtblick/suite";
+import { simpleGetMessagePathDataItems } from "@lichtblick/suite-base/components/MessagePathSyntax/simpleGetMessagePathDataItems";
+import { useLegendCount } from "@lichtblick/suite-base/components/SettingsTreeEditor/useLegendCount";
+
+import { settingsActionReducer, useSettingsTree } from "./settings";
+import type { Config, State, Action } from "./types";
+import { useChartData } from "./useChartData";
+
+type Props = {
+  context: PanelExtensionContext;
+};
+
+const defaultConfig: Config = {
```

Comment:
Create `.../PieChart/constant.ts` file and move all constants there. Please, keep them in upper case.

```suggestion
const DEFAULT_CONFIG: Config = {
```